src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,191 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * The Parser class comprehends and carries out the user commands.
+ * It is used in the Ui class
+ */
+public class Parser {
+    public  Parser(){}
+
+    public static void printLine() {
+        System.out.println("-------------------------------");
+    }
+
+    /**
+     * Constructor for loading deadlines using switch and case.
+     * @param sc            the scanner used to take in the user commands
+     * @param todoList      the array list the stores all the todos
+     * @param store         the Storage that handles the loading and
+     *                      saving of tasks into the schedule text file
+     */
+    public void parse(Scanner sc, ArrayList<Task> todoList, Storage store) throws IOException {
+        String command = sc.next();
+        switch (command) {
+            case "bye":
+                printLine();
```

Comment:
Consider extracting out the lines of code into a function for the case statements to make the code look cleaner 😃